### PR TITLE
VIH-7363 for cacd we are showing 'none' for party in participants added list

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7363
### Change description ###
 - fix for case roles
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts [hunk 1]
```diff
@@ -99,4 +99,11 @@ export class ParticipantsListComponent implements OnInit, OnChanges {
         }
         return interpretee;
     }
+    getCaseName(particpant: ParticipantModel): string {
```
Comment (phillipjchambers) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts:102]: I think you should use an array here & includes, like this:
```
['none','observer','panel member'].includes(participant.case_role_name)
```

Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts:102]: Having said that, I've made some changes in this same area so the point might be moot

Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts:102]: But, generally, when you're doing multiple string comparisons on the same prop, Array.includes is nice

Comment (RocheL) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.ts:102]: yes, will merge with the refactored version.

**Pull request closed without merging**